tests/e2e/cypress.json

```diff
@@ -1,4 +1,5 @@
 {
+  "integrationFolder": "cypress/specs",
```

Comment:
What is this for?

Reply:
This is the property that Cypress uses as the base for searching for specs to run (relative to this `cypress.json` file).

By default the value is `cypress/integration`, in 57bb78f I changed it to `cypress/specs`. Here's more https://docs.cypress.io/guides/references/configuration#Folders-Files